Amazon.QLDB.Driver/result/Result.cs

```diff
@@ -182,19 +182,29 @@ public void Reset()
             /// </summary>
             ///
             /// <returns>The current IOUsage statistics.</returns>
-            internal IOUsage GetConsumedIOs()
+            internal IOUsage? GetConsumedIOs()
             {
-                return new IOUsage(this.readIOs, this.writeIOs);
+                if (this.readIOs != null || this.writeIOs != null)
+                {
+                    return new IOUsage(this.readIOs, this.writeIOs);
```

Comment:
I think that we should make the values of the IOUsage struct and TimingInformation struct no longer nullable, so people dont have to check twice.